patterns/distributed/README.md

```diff
@@ -0,0 +1,160 @@
+# WSO2 API Management Ansible scripts
+
+This repository contains the Ansible scripts for installing and configuring WSO2 API Management.
+
+## Supported Operating Systems
+
+- Ubuntu 16.04 or higher
+- CentOS 7
+
+## Supported Ansible Versions
+
+- Ansible 2.5 or higher
+
+## Directory Structure
+```
+.
+├── dev
+│   ├── group_vars
+│   │   └──apim.yml
+│   ├── host_vars
+│   │   ├── apim-control-plane_1.yml
+│   │   ├── apim-gateway_1.yml
+│   │   └── apim-tm_1.yml
+│   └── inventory
+├── files
+│   ├── lib
+│   │   ├── amazon-corretto-17.0.6.10.1-linux-x64.tar.gz
+│   │   └── mysql-connector-j-8.0.31.jar
+│   └── packs
+│   │   └── wso2am-4.2.0.zip
+│   ├── system
+│   │   └── etc
+│   │       ├── security
+│   │       │   └── limits.conf
+│   │       └── sysctl.conf
+│   └── misc
+├── README.md
+├── roles
+│   ├── apim
+│   │   ├── tasks
+│   │   └── templates
+│   ├── apim-control-plane
+│   │   ├── tasks
+│   │   └── templates
+│   ├── apim-tm
+│   │   ├── tasks
+│   │   └── templates
+│   └── common
+│       └── tasks
+├── scripts
+│   ├── update.sh
+│   └── update_README.md
+└── site.yml
+
+```
+
+Following instructions can be followed to deploy a distributed APIM deployment pattern.
```

Comment:
"Below instructions can be followed" or "Following instructions can be implemented" ?

Reply:
Fixed with https://github.com/wso2/ansible-apim/pull/198